### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by pasha30061**
- [DEMO LINK](https://pasha30061.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://pasha30061.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

File: src/styles/blocks/card.scss [hunk 1]
```diff
@@ -0,0 +1,92 @@
...
+    margin-top: 40px;
+  }
+
+  &__code {
+    margin-bottom: 0;
+    margin-top: 4px;
+    color: $secondary-accent-color;
+    font-size: 10px;
+    line-height: 14px;
+    font-weight: 400;
```
Comment (etojeDenys) [inline src/styles/blocks/card.scss:31]: it's default value, so we can remove it

**Review by etojeDenys — CHANGES_REQUESTED** [review]

**Review by maxim2310 — CHANGES_REQUESTED** [review]
.

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,47 @@
...
+      <img
+        class="card__img"
+        src="images/imac.jpeg"
+        alt="Imac"
+      >
+
+      <h3 class="card__title">APPLE A1419 iMac 27" Retina<br> 5K Monoblock (MNED2UA/A)</h3>
+      <p class="card__code">Product code: 195434</p>
+
+
```
Comment (volodymyr-soltys97) [inline src/index.html:24]:
```suggestion

```

**Review by volodymyr-soltys97 — APPROVED** [review]
Great work 🔥

**Pull request closed without merging**